server/src/infra/repositories/media.repository.ts

```diff
@@ -87,7 +87,10 @@ export class MediaRepository implements IMediaRepository {
         ffmpeg(input, { niceness: 10 })
           .outputOptions(options.outputOptions)
           .output(output)
-          .on('error', reject)
+          .on('error', (err, stdout, stderr) => {
+            console.log(stderr);
```

Comment:
Can we use `Logger` here?

Reply:
Ooh, I forgot you could call it like that without making an instance.